ftl/php/builder.py

```diff
@@ -71,24 +72,22 @@ def _gen_package_tar(self, destination_path):
         tar_path = tempfile.mktemp()
         subprocess.check_call(
             ['rm', '-rf', os.path.join(app_dir, 'vendor')])
-        logging.info('Starting composer install ...')
-        subprocess.check_call(
-            ['composer', 'install', '--no-dev', '--no-scripts'], cwd=app_dir)
-        logging.info('Finished composer install.')
 
-        logging.info('Starting to tar composer packages...')
-        subprocess.check_call(['tar', '-C', tmp, '-cf', tar_path, '.'])
-        logging.info('Finished generating tarfile for composer packages.')
+        with ftl_util.Timing("composer install"):
```

Comment:
For searching later it might be nice to make these descriptors all have _ instead of spaces.

Reply:
done